crates/diesel_models/src/payment_intent.rs

```diff
@@ -57,22 +50,29 @@ pub struct PaymentIntent {
     pub updated_by: String,
     pub surcharge_applicable: Option<bool>,
     pub request_incremental_authorization: Option<RequestIncrementalAuthorization>,
-    pub incremental_authorization_allowed: Option<bool>,
     pub authorization_count: Option<i32>,
     pub session_expiry: Option<PrimitiveDateTime>,
-    pub fingerprint_id: Option<String>,
     pub request_external_three_ds_authentication: Option<bool>,
     pub charges: Option<pii::SecretSerdeValue>,
     pub frm_metadata: Option<pii::SecretSerdeValue>,
     pub customer_details: Option<Encryption>,
-    pub billing_details: Option<Encryption>,
     pub merchant_order_reference_id: Option<String>,
-    pub shipping_details: Option<Encryption>,
     pub is_payment_processor_token_flow: Option<bool>,
     pub shipping_cost: Option<MinorUnit>,
     pub organization_id: common_utils::id_type::OrganizationId,
     pub tax_details: Option<TaxDetails>,
     pub skip_external_tax_calculation: Option<bool>,
+    pub merchant_reference_id: String,
+    pub billing_address: Option<Encryption>,
+    pub shipping_address: Option<Encryption>,
+    pub capture_method: Option<storage_enums::CaptureMethod>,
+    pub authentication_type: Option<common_enums::AuthenticationType>,
+    pub amount_to_capture: Option<MinorUnit>,
+    pub prerouting_algorithm: Option<serde_json::Value>,
```

Comment:
Is prerouting required? and why this needs to a Value? where we are moving to a routing id based approach.

Reply:
Yes, we need this to store the pre routing results when payment methods list call is made